RELEASE_NOTES.md

```diff
@@ -1,3 +1,10 @@
+#### 2.1.0 December dd, 2020
+
+* SQL Azure handling of `datetimeoffset`, `datetime2`, `time` causing `SqlMetaData` constructor exception (#393)
```

Comment:
For me this happens with local SQL Server 2019 Developer Edition also, so I don't think it's specifically Azure.

Reply:
@cmeeren right now this test is passing on CI with an older release:

https://github.com/fsprojects/FSharp.Data.SqlClient/blob/4787180d3db4b4a2bc3a58e99ad5209df7ab36ed/tests/SqlClient.Tests/TVPTests.fs#L203-L209

I haven't checked specifically the 2019 version, I'll double check.